src/foam/mlang/mlang.js

```diff
@@ -2418,10 +2418,10 @@ foam.CLASS({
               s = prop.f(obj).toString().toLowerCase();
             } else if ( this.Date.isInstance(prop) ) {
               s = prop.f(obj).toISOString().toLowerCase();
-            } else if ( ! this.String.isInstance(prop) ) {
-              continue;
             } else {
-              s = prop.f(obj).toLowerCase();
+              s = prop.f(obj);
```

Comment:
Would this be better:
s = String.valueOf(prop.f(obj)).toLowerCase();

Reply:
it doesn't work for me for some reason. but i can investigate